lms/envs/common.py

```diff
@@ -3888,3 +3888,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
 SYSLOG_SERVER = ''
 FEEDBACK_SUBMISSION_EMAIL = ''
 GITHUB_REPO_ROOT = '/edx/var/edxapp/data'
+
+##################### SUPPORT URL ############################
+SUPPORT_HOW_TO_UNENROLL_LINK = 'https://support.edx.org/hc/en-us/articles/115015783627'
```

Comment:
It should be empty here and override it in `production.py` like support site url https://github.com/edx/edx-platform/blob/4ead21566553f336078e54bf16294907b9cd00aa/lms/envs/production.py#L354